src/doc/trpl/glossary.md

```diff
@@ -14,3 +14,26 @@ let z = (8, 2, 6);
 ```
 
 In the example above `x` and `y` have arity 2. `z` has arity 3.
+
+### Abstract Syntax Tree
```

Comment:
Maybe this could include the 'AST' abbreviation (not necessarily in the title, but somewhere).